### ISSUE TITLE
[Papa's Pancakeria, Papa's Wingeria] Breakfast Blast does not work correctly

### ISSUE THREAD

**Issue by InputEvelution** [ruffle-rs/ruffle#18592]
### Describe the bug

In Breakfast Blast, the pancakes/toast/waffles do not move across the screen as expected, instead moving straight up and down across the screen. They are also impossible to hit, making the minigame unbeatable.
https://youtu.be/V2Pj_WxXNiE

### Expected behavior

The pancakes/toast/waffles should appear to fly off into the distance. When clicked on with the crosshair, they should explode, giving the player points.

### Content Location

https://www.flipline.com/games/papaspancakeria/license.html
https://www.flipline.com/games/papaswingeria/license.html

### Affected platform

Desktop app

### Operating system

Windows 10

### Additional information

The console suggests that this bug occurs due to stubbed and unimplemented functions in Ruffle.
![Image](https://github.com/user-attachments/assets/d4da769b-53f9-4d9a-b743-61629c1fb522)
![Image](https://github.com/user-attachments/assets/10670f72-ccc0-4062-8394-db063011a9f8)

**Comment by n0samu** [comment]
Yes this is due to unimplemented 3D DisplayObject properties (DisplayObject.z, etc). Are you able to recheck this? 3D effects still won't be working, but it might be playable now depending on the exact implementation

**Comment by InputEvelution** [comment]
Testing on latest nightly, it is still unplayable, though visually it looks different to my initial report. The pancakes/toast/waffles are no longer visible, and can still no longer be shot, but their shadows can be seen moving along the foreground in sync with their trajectories.
